mesonpy/__init__.py

```diff
@@ -631,7 +631,7 @@ def build_editable(self, directory: Path, verbose: bool = False) -> pathlib.Path
         return wheel_file
 
 
-MesonArgsKeys = Literal['dist', 'setup', 'compile', 'install']
+MesonArgsKeys = Literal['dist', 'setup', 'compile', 'install', 'install-tags']
```

Comment:
I wonder it wouldn't just be better to parse this from the `install` args.

Reply:
good point, I'll implement this
```suggestion
MesonArgsKeys = Literal['dist', 'setup', 'compile', 'install']
```

Reply:
Thanks! This can be done with `argparse` and [`parse_known_args`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.parse_known_args).